.github/workflows/ci-tests.yml

```diff
@@ -1,7 +1,10 @@
-name: "PR CI Validation"
+name: "CI Tests"
 
 on:
-  pull_request
+  schedule:
+    - cron: "0 7 * * *"
```

Comment:
Does the scheduled build need to be configured to only build the `main` branch? We don't need the nightly build to be run on every branch.

Reply:
The documentation for the `schedule` action says that it only runs against your repos default branch, so it should just run against `main` https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule